tox.ini

```diff
@@ -1,8 +1,11 @@
 # For more information about tox, see https://tox.readthedocs.io/en/latest/
 [tox]
 minversion = 3.9.0
-envlist = lint,packaging,py{36,37,38,39},devel,distros
-skipsdist = True
+envlist =
+    lint
+    packaging
+    py{36,37,38,39}
+    py{36,37,38,39}-{devel}
```

Comment:
distros got lost. Replaced by gh actions ?

Reply:
Not really it was never defined, in fact I removed few other unused envs.